### ISSUE TITLE
Weighted CE and BCE

### ISSUE THREAD

**Issue by Shiro-LK** [tensorflow/tensorflow#41684]
**System information**
- TensorFlow version (you are using): 2.0
- Are you willing to contribute it (Yes/No):
probably not

**Describe the feature and the current behavior/state.**
For now, loss function such as BCE or CE (Cross Entropy) can not use a specific weights for each class. For instance this feature is possible on pytorch. The way to do it is to use the class_weight option in the .fit method. However, it seems more appropriate to be able to use it through the loss function. 

**Will this change the current api? How?**
I do not know
**Who will benefit with this feature?**
Everyone, especially when using multiple outpu,  it is not possible to use class weight in that specific case.
**Any Other info.**

**Comment by tilakrayal** [comment]
@Shiro-LK,
`tf.nn.weighted_cross_entropy_with_logits` which tries to compute a weighted cross entropy. Could you please take a look at this official doc [link](https://www.tensorflow.org/api_docs/python/tf/nn/weighted_cross_entropy_with_logits) and let us know if you are looking for the similar feature. Thank you!
